modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -137,6 +143,36 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params<Net>& cfgInputReshape(const std::map<std::string, std::vector<std::size_t>> &&r) {
+        desc.shapes_for_layers_reshape = std::move(r);
+        return *this;
+    }
+
+    Params<Net>& cfgInputReshape(const std::map<std::string, std::vector<std::size_t>> &r) {
+        desc.shapes_for_layers_reshape = r;
+        return *this;
+    }
+
+    Params<Net>& cfgInputReshape(const std::string&& s, const std::vector<size_t>&& r) {
```

Comment:
So, only one letter for variable name, not a good idea :(

Reply:
Fixed.